tests/test_multisetter.py

```diff
@@ -62,4 +63,64 @@ def test_custom_setter_scalar():
         assert np.all(ctx2np(qf_setter.get_values()) == np.array([10., 100., 1000.]))
         assert line['qf1'].length == 10.
         assert line['qf2'].length == 100.
-        assert line['qf3'].length == 1000.
\ No newline at end of file
+        assert line['qf3'].length == 1000.
+
+
+def test_custom_setter_fast_quadrupole():
```

Comment:
I would rename FastQuadrupole in to SimpleThinQuadrupole and FastDipole into SimpleThinBend.
Of course all names in tests, examples, etc. need to follow